ops/Ops/CloudFormation/Options.hs

```diff
@@ -0,0 +1,83 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Ops.CloudFormation.Options
+    ( Options(..)
+    , loadOptions
+    ) where
+
+import Data.Text (Text)
+import qualified Data.Text as T
+
+data Options = Options
+    { oApp :: Text
+    , oEnv :: Text -- Staging, Beta, etc
+    , oDomain :: Text
+
+    -- Scale
+    , oAppsClusterSize :: Text
+    , oAppsInstanceType :: Text
+    , oAppServiceCount :: Integer
+    , oBackendServiceCount :: Integer
+
+    -- Settings & Versions
+    , oClusterLogLevel :: Text
+    , oAppImageTag :: Text
+    , oBackendImageTag :: Text
+
+    -- The only manual pre-requisite
+    , oCertificateARN :: Text
+
+    -- Temporary until RDS/ElastiCache
+    , oDatabaseURL :: Text
+    , oRedisURL :: Text
+
+    -- Secrets
+    , oGitHubAppId :: Text
+    , oGitHubAppKey :: Text
+    }
+
+loadOptions :: Text -> IO Options
+loadOptions _ = pure Options
+    { oApp = "Restyled"
+    , oEnv = "Beta"
+    , oDomain = "beta.restyled.io"
+    , oAppsClusterSize = "1"
+    , oAppsInstanceType = "t2.micro"
+    , oAppServiceCount = 1
+    , oBackendServiceCount = 1
+    , oClusterLogLevel = "DEBUG"
+    , oAppImageTag = "latest"
+    , oBackendImageTag = "latest"
+    , oCertificateARN = "arn:aws:acm:us-east-1:918255682310:certificate/ab8d47ed-d59e-461a-b54f-19febb31993a"
+    , oDatabaseURL = "postgres://wjmxckqnoobjvy:7b1be3bd5c13a28b178e4988f666eaa4a3855ff52ebe10ae97cab941b20cd3d2@ec2-184-73-189-221.compute-1.amazonaws.com:5432/d11cp30so3lfkl"
+    , oRedisURL = "redis://h:pe615abeda2b50f97c124c6e6c5a78d6fd8a8e8084f8d8a2c8a10963c28c74e3e@ec2-34-203-180-66.compute-1.amazonaws.com:64409"
+    , oGitHubAppId = "5518"
+    , oGitHubAppKey = T.unlines
+        [ "-----BEGIN RSA PRIVATE KEY-----"
```

Comment:
This is a key to an app that's only install-able in this org, which holds only public repos, and it has limited access. I've burnt it anyway, because why not, but it's not sensitive.